src/components/App/App.js

```diff
@@ -38,24 +40,26 @@ class App extends Component  {
           <h1>Wilson Picker</h1>
           <nav>
             <button onClick={() => this.generateColors()}>Generate a Palette</button>
+            <Link
+              activeClass="active"
+              to="projects-section"
+              spy={true}
+              smooth={true}
```

Comment:
This is cool. Youll have to explain to me what the spy and smooth props are. good job!